bindings/wasm/examples-account/src/tests/config.ts

```diff
@@ -0,0 +1,11 @@
+import {config} from "../config";
+
+const TIMEOUT = 1000*60*3; // 3 minutes to account for spurious network delays
+
+// Only verifies that no uncaught exceptions are thrown, including syntax errors etc.
+describe("Test node examples", function () {
+    this.timeout(TIMEOUT);
```

Comment:
The timeout can be specified in the script now, so setting the timeout individually is unnecessary now. I think I added them because the CLI option didn't work when they were all in the same file or something? Not sure really.

Applies to all test files, including regular Node.js examples.

```suggestion
// Only verifies that no uncaught exceptions are thrown, including syntax errors etc.
describe("Test node examples", function () {
```